### ISSUE TITLE
Dynamic path with sync-missing-translation-keys

### ISSUE THREAD

**Issue by orblazer** [joedixon/laravel-translation#85]
**Describe the bug**
When i use artisan for generate missing translation keys, this make an invalid key because in my code i have an dynamic translation path.

Currently i have `__('games.game.' . $serverInfo->type)` and this generate in main json file :
```json
{
    "games.game.' . $serverInfo->type) }}\">\n                                {!! App\\Helpers\\MinecraftColor::clean($serverInfo->displayName) !!}\n                            <\/td>\n                            <td class=\"text-right\">\n                                @if (isset($serverInfo->status) && $serverInfo->status->state === 'on": ""
}
```

**To Reproduce**
1. Create an view
2. Create an dynamic translation like `__('games.' . $game)`
3. See error

**Expected behavior**
When use `php artisan translation:sync-missing-translation-keys` don't generate dynamic path if that could not be parsed.
**OR** : Create part of path. exemple if my code is `__('games.game.' . $game)`, this create the array for `games.game` and keep that array empty

**Comment by Eskiell** [comment]
This package is discontinued. Not have suport to larave 6x

**Comment by joedixon** [comment]
Right now, I don't have plans to implement this, but would be open to a PR if you have the time.

**Comment by orblazer** [comment]
Ok @joedixon i could try make it but i need know want behavior should be when make an string based like :
- `__('This is an string with argument (year: ' . $year . ')')` : This keep the current behavior ? Go in json at :
  ```json
  {
    "This is an string with argument (year: ' . $year . ')": ""
  }
  ```
- `__('array.' . $key)` : Should be modifier for go in `array` ? Like :
  ```php
  return array(
    "array" => array()
  );
  ```
- But what behavior when only use `__($message)`, dont generate path ?

I currently work on that behavior.

**Comment by joedixon** [comment]
The problem is, when you create variable keys, there can be lots of different values for that translation.

I think maybe it would be better to present a warning when the package encounters a variable key?

What do you think?

**Comment by orblazer** [comment]
Yes that look like good, show warning like `WARNING ! You use an variable in translation key string at file.php (50:10). You need use directly an variable for translation key`
